js/Utilities.js

```diff
@@ -1341,9 +1335,5 @@ var downloadButton = function(name){
 
 				var xml = (mxUtils.getXml(root));
 
-				new mxXmlRequest(builder_path + "/download.php", $.param({
-					name: "Insight Maker Diagram",
-					"format": "svg",
-					"data": xml
-				})).simulate(document, "_blank");
+				downloadFile("Insight Maker Diagram.svg", xml)
```

Comment:
Please check this this works when serving in the web browser and it's not blocked by an anti-spam security mechanism.

Also semi-colon at end of line.